src/Simplex/Chat/Migrations/M20231219_notes_folders.hs

```diff
@@ -0,0 +1,41 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Chat.Migrations.M20231219_notes_folders where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+m20231219_notes_folders :: Query
+m20231219_notes_folders =
+  [sql|
+    CREATE TABLE notes_folders (
+      notes_folder_id INTEGER PRIMARY KEY AUTOINCREMENT,
+      user_id INTEGER NOT NULL REFERENCES users ON DELETE CASCADE,
+      local_display_name TEXT NOT NULL,
+      created_at TEXT NOT NULL,
+      updated_at TEXT NOT NULL,
+      chat_ts TEXT,
+      favorite INTEGER NOT NULL DEFAULT 0,
+      unread_chat INTEGER DEFAULT 0 NOT NULL,
```

Comment:
doesn't seem to make sense to mark your own notes as unread
```suggestion
```

Reply:
"Dear future me, this folder contains some urgent stuff for you to resolve ASAP"

Reply:
also, to have unified pagination/search/filter code with direct/group

Reply:
unread won't be available in the UI until we support multiple Note folders (we don't need API to create/delete now, it'll be just created for each user profile when profile is created).